docs/roadmap.md

```diff
@@ -17,6 +16,21 @@ Join the chat <a href="https://gitter.im/kics-io/community" target="_blank">on G
 
 | Release | Feature |
 | --- | --- |
+| **August 2021** |
+| v1.4.0 | Azure Resource Manager support |
+| v1.4.0 | Terraform functions |
+| v1.4.0 | CIS descriptions |
+| **July 2021** |
+| v1.3.5 | --input-data option |
+| v1.3.5 | Improved documentation |
+| **June 2021** |
+| v1.3.3 | PDF report |
+| v1.3.2 | Gitlab SAST report |
+| v1.3.2 | Improved HTML report (path, platform, start/end times)|
+| **May 2021** |
+| v1.3.0 | Improved Logs (CPU, RAM outputs) |
```

Comment:
feat(performance): concurrent engine scans by parser (#3085) (#3061)
Is one of the big changes for 1.3.0